delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/realms/Realm.scala

```diff
@@ -65,4 +79,7 @@ object Realm {
   val context: ContextValue                             = ContextValue(contexts.realms)
   implicit val realmJsonLdEncoder: JsonLdEncoder[Realm] =
     JsonLdEncoder.fromCirce(context)
+
+  implicit val metadataEncoder: Encoder.AsObject[Metadata]    = deriveConfiguredEncoder[Metadata]
```

Comment:
I think this one should be private and only the `JsonLdEncoder[Metadata]` public